pkg/core/chain/chain.go

```diff
@@ -489,9 +490,18 @@ func (c *Chain) runStateTransition(tipBlk, blk block.Block) (*block.Block, error
 	// Update the provisioners.
 	// blk.Txs may bring new provisioners to the current state
 	c.p = &provisionersUpdated
+	provNumAfter := c.p.Set.Len()
+
+	// Print provisionres information if provisioner(s) have been
+	// added/removed with this block acceptance
+	if provNumAfter-provNumBefore != 0 || blk.Header.Height == 1 {
```

Comment:
Did you consider the case when there is a provisioner removed and another one added in the same block?
Ps: this can only happen when changing epoch

Reply:
Good point. This could also happen if a provisioner unstakes. Maybe the approach should be revised then. I'll think about it.

Reply:
It turns out that we already print provisioners list in consensus if debug level is set, so there is not need to pollute Info log level with duplicated details.